src/components/AddTodo.js

```diff
@@ -0,0 +1,42 @@
+import React, { useState } from 'react';
+import { useDispatch } from "react-redux";
+import { addTask } from "../redux/tasksSlice";
```

Comment:
always use absolute paths
```suggestion
import { addTask } from "store/redux/tasksSlice";
```